apps/fishing-map/features/user/UserLoggedIconButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useSelector } from 'react-redux'
+import { IconButton, IconButtonProps } from '@globalfishingwatch/ui-components'
+import LocalStorageLoginLink from 'routes/LoginLink'
+import { isGuestUser } from 'features/user/user.slice'
+
+type UserLoggedIconButton = IconButtonProps & { loginTooltip?: string }
+
+const UserLoggedIconButton = (props: UserLoggedIconButton) => {
+  const { t } = useTranslation()
+  const [isLoginHover, setIsLoginHover] = useState(false)
+  const guestUser = useSelector(isGuestUser)
+
+  if (guestUser) {
+    return (
+      <LocalStorageLoginLink>
+        <IconButton
+          {...props}
+          icon={isLoginHover ? 'user' : props.icon}
+          tooltip={
+            props.loginTooltip ||
+            t('vessel.infoLogin', 'Register and login to see more details (free, 2 minutes)')
+          }
+          onMouseEnter={() => setIsLoginHover(true)}
+          onMouseLeave={() => setIsLoginHover(false)}
+        />
```

Comment:
We would need to avoid passing `onClick` handler to the `IconButton` to avoid the download to be executed when clicking (and only triggering the login link action). Maybe passing undefined as `onClick` value?

 ```suggestion
          onClick={undefined}
        />
```

Reply:
Great catch! fixed [here](https://github.com/GlobalFishingWatch/frontend/pull/2364/commits/11e6ddea1960cd67b8dbb5a717d9b2f6df2847d2)